src/app/components/pages/IsaacCompetition/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { memo } from "react";
+import { Card, CardBody, CardHeader, Collapse } from "reactstrap";
+
+interface AccordionItemProps {
+  id: string;
+  title: string;
+  section: string[];
+  open: string | null;
+  setOpenState: (id?: string) => void;
+}
+
+const renderSectionContent = (section: string[]) => (
+  <>
+    {section[0]}
+    <ul>
+      {section.slice(1, -1).map((item) => (
+        <li key={item}>{item}</li>
+      ))}
+    </ul>
+    {section[section.length - 1]}
+  </>
+);
+
+const AccordionItem = memo(({ id, title, section, open, setOpenState }: AccordionItemProps) => {
+  const toggle = (id: string) => {
+    setOpenState(open === id ? undefined : id);
+  };
+
+  const headerClasses = [
+    "accordion-button p-3 m-0 d-flex justify-content-between",
+    id === "0" && "rounded-top",
+    id === "5" && "rounded-bottom border-bottom-0",
+    open !== id && "border-bottom border-dark",
+  ]
+    .filter(Boolean)
+    .join(" ");
+
+  return (
+    <Card eventKey={id}>
+      <CardHeader className={headerClasses} onClick={() => toggle(id)}>
+        {title}
+        {open === id ? (
+          <img src={"/assets/chevron_up.svg"} alt="Up Icon" className="" />
```

Comment:
we need to work on this chevron slightly and how it's positioned in relation to the text, you can see it's sitting a bit too close on your screenshots.